src/pages/components/tabs-extended.mdx

```diff
@@ -43,13 +43,29 @@ On mobile, Tabs extended switches to an accordion for easier browsing.
 <Row>
 <Column colMd={6} colLg={6}>
 
-![Image of Tabs extended component on `sm`](../../images/component/tabs-extended/components--tabs-extended-2.png)
+![Image of Tabs extended component on `sm`](../../images/component/tabs-extended/components--tabs-extended-3.png)
 
 <Caption>Example of the tabs extended, on mobile</Caption>
 
 </Column>
 </Row>
 
+### Modifiers
+
+#### Orientation
+
+Tabs orientation on desktop can be set to horizontal or vertical. When using vertical, the tabs are displayed stacked in 1 column. This orientation allows for longer tab labels.
```

Comment:
I would rework this a little bit:

"Tabs orientation on desktop can be set to horizontal or vertical. When using the vertical orientation, the tabs are displayed stacked in one column. This allows for longer tab labels."

Reply:
I like your rework, @RichKummer